kili/utils.py

```diff
@@ -1,11 +1,13 @@
 """
 Utils
 """
-from typing import List, Callable
-
+from typing import Dict, List, Callable, Optional
 import time
 from tqdm import tqdm
 
+from .helpers import GraphQLError
+from .constants import MUTATION_BATCH_SIZE, THROTTLING_DELAY
+
 # pylint: disable=too-many-arguments,too-many-locals
 
 
```

Comment:
Transform this utils file into a folder, move the pagination stuff in `pagination.py`